docs/content/installation/integrations/app-protect-waf/configuration.md

```diff
@@ -123,7 +123,7 @@ For example, say you want to [log state changing requests](/nginx-app-protect-wa
     "content": {
         "format": "default",
         "max_request_size": "any",
-        "max_message_size": "5k"
+        "max_message_size": "64k"
```

Comment:
```suggestion
      "max_request_size": "2k",
      "max_message_size": "5k"
```

Reply:
Just to be clear that is the value suggested by NAP team 2k